ssl/statem/statem_lib.c

```diff
@@ -277,11 +277,8 @@ MSG_PROCESS_RETURN tls_process_cert_verify(SSL *s, PACKET *pkt)
 {
     EVP_PKEY *pkey = NULL;
     const unsigned char *data;
-#ifndef OPENSSL_NO_GOST
-    unsigned char *gost_data = NULL;
-#endif
```

Comment:
I'm sorry, I misled you regarding this one. There's now an `OPENSSL_free` further down that's missing its variable...